uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -2478,8 +2478,12 @@ public function testEmailGetsSentWhenLeaveRequestIsCreatedAndUpdated() {
 
     $this->setContactAsLeaveApproverOf($manager1, $leaveContact);
 
+    $absenceType = AbsenceTypeFabricator::fabricate([
+      'calculation_unit' => '1'
```

Comment:
No need to pass in the calculation_unit parameter, the Fabricator already passes that and defaults it to 1.

Reply:
Done.